test-openshift.yaml

```diff
@@ -51,7 +51,12 @@
           shell: oc project {{ openshift_project_name }}
 
         - name: Import the image into OpenShift
-          shell: oc import-image {{ image_name }} --from={{ image_full_name }} --insecure=true --confirm
+          shell: oc import-image {{ image_name }}:{{ environment[0]['VERSION'] }} --from={{ image_full_name }} --insecure=true --confirm
+          retries: 3
+          delay: 10
+
+        - name: Tag image into OpenShift
+          shell: oc tag {{ image_name }}:{{ environment[0]['VERSION'] }} nodejs:{{ environment[0]['VERSION'] }}
```

Comment:
Do we want to tag every container image as `nodejs`?

Reply:
This is a great catch. Definitely not.

Reply:
Fixed by https://github.com/sclorg/container-common-scripts/pull/280/commits/0498bcb4329668ccbb026795e99315932021ff83